db/structure.sql

```diff
@@ -3289,6 +3290,13 @@ CREATE UNIQUE INDEX index_assignment_properties_on_assessment_id ON public.assig
 CREATE INDEX index_assignment_properties_on_default_starter_file_group_id ON public.assignment_properties USING btree (default_starter_file_group_id);
 
 
+--
+-- Name: index_assignment_properties_on_token_end_date; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE INDEX index_assignment_properties_on_token_end_date ON public.assignment_properties USING btree (token_end_date);
```

Comment:
This should no longer exist, since you removed it from the migration. This indicates that this file is out of sync with your migration. The easiest way to fix this is to revert all changes in this specific file, then re-run the migration.